### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by BraisGabin**
Right now our gradle documentation is really verbose It repeats the same block of code 5 times so it's difficult to find what you are looking for.

I do think that jcenter is not the best repository and qw should be avoid it when possible. But that's the user choice and our documentation is not the place where we should promote that.

### CODE REVIEW

**Comment by BraisGabin** [comment]
We need to do something with `codecov` it's failing all the time.

**Review by schalkms — APPROVED** [review]

File: docs/pages/gettingstarted/gradle.md [hunk 1]
```diff
@@ -68,18 +68,7 @@ plugins {
...
-    jcenter()
-
-    // or
-
-    mavenCentral()
-    jcenter {
-        content {
-            // just allow to include kotlinx projects
-            // detekt needs 'kotlinx-html' for the html report
-            includeGroup "org.jetbrains.kotlinx"
```
Comment (arturbosch) [inline docs/pages/gettingstarted/gradle.md:80]: hm, I would really like to leave one of these for reference on.

Comment (cortinico) [reply docs/pages/gettingstarted/gradle.md:80]: Agree. Maybe we can refactor this into a `## Repositories` section and have it listed only once?

Comment (BraisGabin) [reply docs/pages/gettingstarted/gradle.md:80]: I implemented the @cortinico suggestion. What do you think?

**Review by arturbosch — APPROVED** [review]

**Pull request merged**